<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowArticles</class>
 <widget class="QMainWindow" name="MainWindowArticles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>629</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>174</red>
        <green>228</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>174</red>
        <green>228</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>254</red>
        <green>229</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>184</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>174</red>
        <green>228</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>174</red>
        <green>228</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>254</red>
        <green>229</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>184</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>174</red>
        <green>228</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>174</red>
        <green>228</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>184</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>184</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>106</red>
        <green>156</green>
        <blue>253</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>3</red>
        <green>52</green>
        <blue>149</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindowArticles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_IDa">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Type">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Couleur">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Quantite">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_PrixUnitaire">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Marque">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Vendre">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exporter">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exporter</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/download-pdf_6348573.png</normaloff>../../../../Downloads/download-pdf_6348573.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Vendre">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IDa_vendu">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IDa_supprime">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>430</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Supprimer">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>430</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/remove_7695824.png</normaloff>../../../../Downloads/remove_7695824.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Ajouter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Afficher">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Modifier">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Top Marques</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/market-penetration_15178126.png</normaloff>../../../../Downloads/market-penetration_15178126.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>450</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/received_545354601345279.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Couleur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantité</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Marque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>430</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Liste des articles epuisés</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Trier">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Trier</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/swap_5296648.png</normaloff>../../../../Downloads/swap_5296648.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Rechercher">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/search_4180975.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImageDisplay">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Previous">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Next">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Refresh">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualiser</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
